<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1681</width>
    <height>967</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>880</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ВД16-Сервис</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Взаимодействие с датчиком</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="8" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>IP адрес ВД16 (4)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>IP адрес ВД16 (3)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="5">
         <widget class="QSpinBox" name="delay_spin">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string>192.168.1.21</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_15">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Время измерения:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="8">
         <widget class="QLineEdit" name="lineEdit_RMSvalue">
          <property name="text">
           <string>10.00</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>IP адрес ВД16 (2)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number2">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QRadioButton" name="checkBoxSlowly">
          <property name="text">
           <string>Проверка пакетов</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number3">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="7">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>IP ВД16 для калиб-ки или откл-я</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="7">
         <widget class="QLabel" name="label_11">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>СКЗ [м/с^2]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>сек.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>IP адрес ВД16 (1)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="8">
         <widget class="QLineEdit" name="lineEdit_ip_calibration">
          <property name="text">
           <string>192.168.1.21</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="7">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Корректирующий коэффициент</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number4">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Интервал между пакетами</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QSpinBox" name="spinBoxSec">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>21</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="4" column="8">
         <widget class="QLineEdit" name="lineEdit_calibration_value">
          <property name="text">
           <string>0.000800</string>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QRadioButton" name="checkBoxIgnore">
          <property name="text">
           <string>Без проверки пакетов</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number5">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>IP адрес ВД16 (5)</string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number6">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="8">
         <widget class="QPushButton" name="pushButton_poweroff">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 181, 195);</string>
          </property>
          <property name="text">
           <string>Отключить ВД16</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="9">
         <widget class="QTextEdit" name="testConsole"/>
        </item>
        <item row="15" column="0">
         <widget class="QPushButton" name="pushButton_console_clear">
          <property name="text">
           <string>Очистить консоль</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QPushButton" name="pushButton_enable_widgets">
          <property name="text">
           <string>Перезапуск программы</string>
          </property>
         </widget>
        </item>
        <item row="8" column="7">
         <widget class="QLabel" name="label_13">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Пароль для калибровки</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_7">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_5">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="8">
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_6">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="8" column="6">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>мсек</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>IP адрес ВД16 (7)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="4" rowspan="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Получить выборку</string>
          </property>
         </widget>
        </item>
        <item row="9" column="8">
         <widget class="QPushButton" name="pushButton_corr">
          <property name="text">
           <string>Получить кор-й коэффициент</string>
          </property>
         </widget>
        </item>
        <item row="10" column="4">
         <widget class="QPushButton" name="pushButton_RMSvalue">
          <property name="text">
           <string>Получить СКЗ</string>
          </property>
         </widget>
        </item>
        <item row="11" column="8">
         <widget class="QPushButton" name="pushButton_calibration">
          <property name="text">
           <string>Калибровка по СКЗ</string>
          </property>
         </widget>
        </item>
        <item row="10" column="8">
         <widget class="QPushButton" name="pushButton_write_corr">
          <property name="text">
           <string>Калибровка по коэффициенту</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>IP адрес ВД16 (6)</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>IP адрес ВД16 (8)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number7">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QLineEdit" name="lineEdit_ip_tuts_test_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number8">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QCheckBox" name="checkBox_nonstop">
          <property name="text">
           <string>Режим постоянного запроса</string>
          </property>
         </widget>
        </item>
        <item row="15" column="8">
         <widget class="QPushButton" name="pushButton_save_settings">
          <property name="text">
           <string>Сохранить настройки</string>
          </property>
         </widget>
        </item>
        <item row="15" column="7">
         <widget class="QPushButton" name="pushButton_reset_settings">
          <property name="text">
           <string>Сброс настроек</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="lineEdit_device_number1">
          <property name="text">
           <string>0001</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="15" column="4">
         <widget class="QPushButton" name="pushButton_tracking">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Слежение за последней строкой</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>testConsole</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
